opentelemetry-proto/Cargo.toml

```diff
@@ -29,7 +29,7 @@ path = "tests/json_deserialize.rs"
 
 
 [features]
-default = ["full"]
+populate-logs-event-name = []
```

Comment:
is there an easy way to document the feature-flags in general? For this one, we'd like to document that this is experimental, and the attribute name is "name", and is subject to change.

Reply:
We can document in README.md for this crate, or else in Cargo.toml itself, as done by serde.rs - https://github.com/serde-rs/serde/blob/master/serde/Cargo.toml

Reply:
https://docs.rs/opentelemetry-otlp/latest/opentelemetry_otlp/#feature-flags Lets add it here itself along with rest of other flags.

Reply:
done.